includes/reader-revenue/woocommerce/class-woocommerce-logs.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Log WooCommerce events
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class for logging WooCommerce events.
+ */
+class WooCommerce_Logs {
+	/**
+	 * Initialize the class.
+	 */
+	public static function init() {
+		add_filter( 'woocommerce_add_error', [ __CLASS__, 'log_error_notices' ] );
+	}
+
+	/**
+	 * Log error notices.
+	 *
+	 * @param string $message The error message.
+	 */
+	public static function log_error_notices( $message ) {
```

Comment:
We may want to bail if `$message` is empty, otherwise it doesn't tell us much.

Reply:
Good call! Added in b128b89141507180e1b42f0700825de3ee95d396